setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='omnibus',
-    version='1.0.0',
+    version='1.0.1',
```

Comment:
This definitely feels like a 1.1 bump not a 1.0.1 bump. The API is changing and functionality could be considered changing too.